### PR TITLE
feat: Create add space page.

### PR DESCRIPTION
**Pull request by deepakvenkat**
**What**
---
Creates add space page along with the required sub pages. 
Integration to backend to follow in a separate pr. 

**Clubhouse**
---

- [117](https://app.clubhouse.io/out-in-tech/story/117)
- [118](https://app.clubhouse.io/out-in-tech/story/118)
- [119](https://app.clubhouse.io/out-in-tech/story/119)
- [120](https://app.clubhouse.io/out-in-tech/story/120)
- [121](https://app.clubhouse.io/out-in-tech/story/121)
- [122](https://app.clubhouse.io/out-in-tech/story/122)

**Changes**
----
* Adds a new dependency `@material-ui/lab` to use the `Alert` component
* Adds a new route `AddSpace` 
* Adds a new component `AddSpacePage` with each of the step in the add space as a sub component. 
* Hooks up data flow between the different steps in the addspace page. 
* Adds some mock data for business, yelp results, and chips.

### CODE REVIEW

File: src/api/states.js [hunk 1]
```diff
@@ -0,0 +1,230 @@
+export default [
```
Comment (deepakvenkat) [inline src/api/states.js:1]: Hardcoded list of US state and territories to display in the state dropdown.

File: src/api/chips.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
+export default [{
```
Comment (deepakvenkat) [inline src/api/chips.js:1]: Hardcoding this. It can come from our enum table in the future.

File: src/components/AddSpacePage/Address.jsx [hunk 3]
```diff
@@ -0,0 +1,115 @@
+import React, { useState } from 'react';
```
Comment (deepakvenkat) [inline src/components/AddSpacePage/Address.jsx:1]: image

File: src/components/AddSpacePage/Attributes.jsx [hunk 4]
```diff
@@ -0,0 +1,112 @@
+import React, { useState } from 'react';
```
Comment (deepakvenkat) [inline src/components/AddSpacePage/Attributes.jsx:1]: image

File: src/components/AddSpacePage/Review.jsx [hunk 5]
```diff
@@ -0,0 +1,175 @@
+import React, { useState } from 'react';
```
Comment (deepakvenkat) [inline src/components/AddSpacePage/Review.jsx:1]: image

File: src/components/AddSpacePage/Search.jsx [hunk 6]
```diff
@@ -0,0 +1,240 @@
+import React, { useState } from 'react';
```
Comment (deepakvenkat) [inline src/components/AddSpacePage/Search.jsx:1]: image

File: src/components/AddSpacePage/Submit.jsx [hunk 7]
```diff
@@ -0,0 +1,167 @@
+import React, { useState } from 'react';
```
Comment (deepakvenkat) [inline src/components/AddSpacePage/Submit.jsx:1]: image

File: src/components/BusinessResultCard/BusinessResultCard.jsx [hunk 8]
```diff
@@ -0,0 +1,75 @@
...
+  wrapper: {
+    display: 'flex',
+    '& h6': {
+      fontWeight: 600,
+      marginRight: 15,
+    },
+  },
+});
+
+const BusinessResultCard = ({
```
Comment (deepakvenkat) [inline src/components/BusinessResultCard/BusinessResultCard.jsx:38]: This is to display the results from yelp. 

image

**Review by InManuBytes — APPROVED** [review]

File: src/__mocks__/getYelpResultMock.js [hunk 9]
```diff
@@ -0,0 +1,15 @@
+const getYelpResultMock = (overrides = {}) => {
+  const business = {
+    id: '123',
+    name: 'La colombe coffee roasters',
+    category: 'Coffee',
```
[the remark this answers is not in the document]
Comment (deepakvenkat) [reply src/__mocks__/getYelpResultMock.js:5]: Thanks. I have added a note to the integration ticket https://app.clubhouse.io/out-in-tech/story/124/integration-add-space-send-data-from-fe-to-be-all-pages

**Pull request merged**